src/figuresClasses.ts

```diff
@@ -2,10 +1,76 @@
+enum ShapeType {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum FigureColor {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+export interface Figure {
+  shape: ShapeType;
+  color: FigureColor;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = ShapeType.Triangle;
 
-export class Triangle implements Figure {}
+  constructor(
+    public color: FigureColor,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = ShapeType.Triangle;
+  }
 
-export class Circle implements Figure {}
+  getArea(): number {
+    const halfPerimeter = (this.a + this.b + this.c) / 2;
+
+    return +Math.sqrt(
+      halfPerimeter *
+        (halfPerimeter - this.a) *
+        (halfPerimeter - this.b) *
+        (halfPerimeter - this.c),
+    ).toFixed(2);
+  }
+}
 
-export class Rectangle implements Figure {}
+export class Circle implements Figure {
+  public shape;
+
+  constructor(
+    public color: FigureColor,
+    public radius: number,
+  ) {
+    this.shape = ShapeType.Circle;
+  }
+
+  getArea(): number {
+    return +(3.14 * this.radius ** 2).toFixed(2);
+  }
+}
+
+export class Rectangle implements Figure {
+  public shape;
+
+  constructor(
+    public color: FigureColor,
+    public width: number,
+    public heigh: number,
```

Comment:
There is a typo in the parameter name 'heigh'. It should be 'height' to correctly represent the dimension of the rectangle.